doc/REPRODUCIBLE.md

```diff
@@ -146,6 +148,8 @@ repository (remember to checkout the tag you are trying to build):
 
 ```bash
 sudo docker run --rm -v $(pwd):/repo -ti cl-repro-bionic
```

Comment:
maybe this is a change that need to be done inside the docker container, but during the the process that start in this line I need to ran `mkdir release` and rerun the process again.

Maybe can be done inside the docker container or at least documented it?